v2_app.py

```diff
@@ -46,9 +46,8 @@ def get_issues_by_owner_id_v2(owner, issue):
             # week_avg ,cont_name,cont_id,w_goal,w_learn,weekby_avgs,org_link = find_week_avg(issue_url)
             # mentors_data = find_mentors(val['issue_url']) if val['issue_url'] else {'mentors': [], 'mentor_usernames': []}
             
-            if val['body_text']:
-                # and ("@"+val['created_by'].lower() == dmp_issue_id['mentor_username'].lower())
-                if ("Weekly Goals" in val['body_text'] and not w_goal_url):
+            if val['body_text']:                                
```

Comment:
Please optimise this.

Reply:
i think this is optimized, still tried but the only change we can make is to take out val['body_text'] as body_text = val['body_text'], other than this the code is optimized.